### DISCUSSION TITLE
Hola, tengo un problema, tengo mi api en vercel y al entrar si me muestra que esta encendida, pero al hacer peticiones no funciona, y el mismo codigo de la misma api si me funciona en localhost, y no es problema del cors porque tengo el acceso permitido para todos los origenes: https://classify-api-flax.vercel.app/

### DISCUSSION THREAD

**Discussion by bushcoding** [vercel/community#7400] [Account Issue]
### Question

Hola, tengo un problema, tengo mi api en vercel y al entrar si me muestra que esta encendida, pero al hacer peticiones no funciona, y el mismo codigo de la misma api si me funciona en localhost, y no es problema del cors porque tengo el acceso permitido para todos los origenes: https://classify-api-flax.vercel.app/

**Comment by pawlean** [comment]
Hi @bushcoding!

Here are some things to try:
1. Review the logs for any errors or warnings in [Vercel Logs](https://vercel.com/docs/observability/logs).
2.Ensure the URL used for requests is correct and verify API Endpoint.
3. Confirm all required environment variables are correctly set in Vercel.
4. Double-check your CORS settings to allow requests from all origins.
5. Ensure the build and deployment processes are error-free; try re-deploying.
6. Verify you are not hitting Vercel's rate limits.

Let us know how you get on!

[a comment by an automated account is not shown]
